pkg/utils/bodystring.go

```diff
@@ -18,14 +18,14 @@ package utils
 
 import (
 	"errors"
-	"io"
 	"io/ioutil"
 	"net/http"
+	"strings"
 )
 
 // Return the body from a response as a string
 func GetStringFromResponse(r *http.Response) (string, error) {
-	body, err := ioutil.ReadAll(io.LimitReader(r.Body, r.ContentLength))
+	body, err := ioutil.ReadAll(r.Body)
```

Comment:
We need the limit to make sure we do not read forever.  So this line does not need to be fixed.